EasyPost.Tests/AddressTest.cs

```diff
@@ -1,147 +1,127 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EasyPost.Models.API;
+using Xunit;
 
 namespace EasyPost.Tests
 {
-    [TestClass]
-    public class AddressTest
+    public class AddressTest : UnitTest
     {
-        private TestUtils.VCR _vcr;
-
-        [TestInitialize]
-        public void Initialize()
-        {
-            _vcr = new TestUtils.VCR("address");
-        }
-
-        private static async Task<Address> CreateBasicAddress()
-        {
-            return await Address.Create(Fixture.BasicAddress);
-        }
-
-        [TestMethod]
-        public async Task TestCreate()
+        public AddressTest() : base("address")
         {
-            _vcr.SetUpTest("create");
-
-            Address address = await CreateBasicAddress();
-
-            Assert.IsInstanceOfType(address, typeof(Address));
-            Assert.IsTrue(address.id.StartsWith("adr_"));
-            Assert.AreEqual("388 Townsend St", address.street1);
         }
 
-        [TestMethod]
-        public async Task TestCreateVerify()
+        [Fact]
+        public async Task TestAll()
         {
-            _vcr.SetUpTest("create_verify");
+            UseVCR("all");
 
-            Dictionary<string, object> addressData = Fixture.IncorrectAddressToVerify;
-            addressData.Add("verify", true);
+            AddressCollection addressCollection = await Client.Address.All(new Dictionary<string, object?>
+            {
+                {
+                    "page_size", Fixture.PageSize
+                }
+            });
 
-            Address address = await Address.Create(addressData);
+            List<Address> addresses = addressCollection.addresses;
 
-            Assert.IsInstanceOfType(address, typeof(Address));
-            Assert.IsTrue(address.id.StartsWith("adr_"));
-            Assert.AreEqual("417 MONTGOMERY ST FL 5", address.street1);
+            Assert.True(addresses.Count <= Fixture.PageSize);
```

Comment:
Issue: It appears some of your assertions have disappeared (please assert it's `true`):

`Assert.IsNotNull(addressCollection.has_more);`

I think maybe this is true of many of the tests. Please 1:1 double check every test, I'd rather not dig deep to make sure every assertion matches what was there before, that should implicitly be the expectation that the test assertions perfectly match what is on master and what is making it into your PR.

Reply:
Resolved.